### ISSUE TITLE
Use JSDoc to document an ES6 class property

### ISSUE THREAD

**Issue by JohnnyVM** [kkoomen/vim-doge#638]
**Is your feature request related to a problem? Please describe.**
Hello
I would like to suggest use JSDoc type for document ES6 class properties
If you agree and you don't have time, I can try to implement it myself and PR.

**Describe the solution you'd like**
this is the expected generated documentation
```js
class Foo {
// documentation generated
  /**
   * [TODO:description] 
   * @type {[TODO:type]}
   */
  property // use the command DogeGenerate here
}
```

**Describe alternatives you've considered**
other alternatives are described here:
https://stackoverflow.com/questions/51467835/how-to-use-jsdoc-to-document-an-es6-class-property?rq=3

**Comment by kkoomen** [comment]
Hey,

Yeah sure, I'll implement it this weekend.

**Comment by JohnnyVM** [comment]
A lot of thanks :)

**Comment by kkoomen** [comment]
@JohnnyVM I forgot to mention: this feature has been merged and released in [v4.6.0](https://github.com/kkoomen/vim-doge/releases/tag/v4.6.0).

Feel free to submit any new issues if you experience any unwanted behavior in the future. Thanks for your contribution.
